src/directives/hover/README.md

```diff
@@ -84,4 +84,4 @@ In the following, we are swapping icons and tet color depending on the hover sta
 ## Accessibility concerns
 
 Hover state should not be used to convey special meaning, as screen reader users and keyboard only
-users typically ac not typically trigger hover state on elements.
+users typically do not trigger hover state on elements.
```

Comment:
Actually this was supposed to be `users typically can not trigger`... could you update it?